.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Remind me, did we choose to use the `pre-commit` file to control Husky, or the code block in `package.json`? I think you only need one or the other and I'm seeing both (but I might be mistaken - it's been a hot minute since I've worked on this one).

Reply:
the pre-commit file